pyproject.toml

```diff
@@ -16,49 +16,50 @@ autopep8 = "1.5.4"
 certifi = "2020.6.20"
 chardet = "3.0.4"
 colorclass = "2.2.0"
-Django = "^3.2"
+Django = "^4.2"
 django-appconf = "1.0.4"
-django-appdata = "0.3.2"
+django-appdata = "0.4.0"
 django-auth-ldap = "2.2.0"
 django-classy-tags = "2.0.0"
 django-cms = "^3.8.0"
 django-filer = "^2.2"
 django-formtools = "2.2"
-django-haystack = "3.0"
+django-haystack = "3.2.1"
 django-ipware = "3.0.1"
 django-js-asset = "1.2.2"
 django-meta = "^2.1"
 django-mptt = "0.14.0"
-django-parler = "2.2"
+django-parler = "2.3"
 django-polymorphic = "3.0.0"
 django-sekizai = "2.0.0"
 django-settings-export = "1.2.1"
 django-sortedm2m = "3.0.2"
 django-spurl = "0.6.7"
 django-standard-form = "1.1.1"
-django-taggit = "1.2.0"
-django-taggit-autosuggest = "0.3.8"
+django-taggit = "4.0.0"
+django-taggit-autosuggest = "0.4.1"
 django-taggit-templatetags = "0.2.5"
 django-templatetag-sugar = "1.0"
 django-treebeard = "^4.4"
 djangocms-admin-style = "2.0.0"
 djangocms-apphook-setup = "0.4.1"
 djangocms-attributes-field = "2.1.0"
 djangocms-blog = "^1.2"
-djangocms-bootstrap4 = "3.0.0"
+# djangocms-bootstrap4 commit to remove deprecated imports: https://github.com/django-cms/djangocms-bootstrap4/pull/162
+djangocms-bootstrap4 = { git = "https://github.com/django-cms/djangocms-bootstrap4", rev = "2fc7f14c2b7b53882faf6830289b5052f775965f" }
 djangocms-column = "^2.0"
 djangocms-file = "3.0.0"
-djangocms-forms-maintained = { git = "https://github.com/avryhof/djangocms-forms", rev = "d8a69efd2f447ee2f940c7b6f5b8b088c9cb79ed" }
+djangocms-forms-maintained = { git = "https://github.com/TACC/djangocms-forms", rev = "95489d535b8e03c98fcbf9a3d848f1139c896241" }
 djangocms-googlemap = "2.0.0"
 djangocms-icon = "2.0.0"
 djangocms-link = "^3.1"
-djangocms-page-meta = "^1.1.0"
+djangocms-page-meta = "1.3.0"
```

Comment:
Even though [v1.4.1](https://github.com/nephila/djangocms-page-meta/releases/tag/1.4.1) fixes [the bug](https://github.com/nephila/djangocms-page-meta/issues/173) that required removing `^`, I want to continue to keep it off, for now.

<details><summary>Why?</summary>

That the simple bug occurred makes me want to continue to pin this, for safety. I can revisit the upgrade in isolation after this PR is done (for its [one feature](https://github.com/nephila/djangocms-page-meta/pull/169)).

</details>